pypeit/wavetilts.py

```diff
@@ -72,6 +86,7 @@ def __init__(self, coeffs, nslit, spat_id, spat_order, spec_order, func2d, bpmti
         datamodel.DataContainer.__init__(self, d=d)
 
     def _init_internals(self):
+        self.filename = None
```

Comment:
I thought all references to "master" were purged in @kbwestfall PR? Are you not branched off of that branch?